docs/services-web-app.md

```diff
@@ -12,3 +29,255 @@ access to private repos and increases API quotas. To set up a personal access to
 1. Select all repo scopes
 1. Copy the `/services/web-app/.env.example` file and rename to `/services/web-app/.env.local`
 1. Add your token to that new `.env.local` file
+
+## Dependencies Set up
+
+### Run Mode
+
+The app uses run-mode to make decisions that are environment-dependant (cookies, next-server...).
+This package configuration is also needed for proper functionality of the Auth Package.
+
+Make sure required environment variables are configured for proper functionality of this package.
+(See [run-mode package](./packages-api.md#run-mode) for details on how to set up the run-mode
+package)
+
+_Note_: this package requires different values depending on environemnt, make sure to update these
+when running on dev/test/production mode
+
+### Auth
+
+This service depends on the auth package; Make sure required environment variables are configured
+for proper functionality of this package. (See [auth package](./packages-api.md#auth) for details on
+how to set up the auth package)
+
+_Note_: this package requires different environment vairables and values depending on environemnt,
+make sure to update these when running on dev/test/production mode
+
+## Adding Local Certificates
+
+The application must run on https for IBMid authentication to work properly. For such purposes,
+local certificates must be generated to run on development environment. You may use
+[mkcert](https://github.com/FiloSottile/mkcert#installation) tool for this. With the tool
+downloaded, the certificates will be automatically generated next time you
+[run the app securely](#running-app-securely)
```

Comment:
```suggestion
In PROD run mode, the application must run on https for IBMid authentication to work properly. For such purposes,
local certificates must be generated to use authentication in the web-app when running locally in PROD mode.
The [mkcert](https://github.com/FiloSottile/mkcert#installation) tool can help generate these certificates and is used implicitly by the `srart-secure` node script of the web-app. With the tool downloaded, the certificates will be automatically generated the next time you
[run the app securely](#running-app-securely).
```

Reply:
Actually question about this.. Did we ever get to the point where you can use auth in DEV mode without having to have certificates set up? Like do we have the ability to "fake" a login? If not, we should make an issue and aim to address that pretty soon

Reply:
tracking in #212